packages/expo-updates/ios/EXUpdates/Logging/UpdatesErrorCode.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import Foundation
+
+/**
+ Error codes for expo-updates logs
+ */
+@objc(EXUpdatesErrorCode)
+public enum UpdatesErrorCode: Int {
+  case None = 0
+  case NoUpdatesAvailable = 1
+  case UpdateAssetsNotAvailable = 2
+  case UpdateServerUnreachable = 3
+  case UpdateHasInvalidSignature = 4
+  case UpdateFailedToLoad = 5
+  case AssetsFailedToLoad = 6
+  case JSRuntimeError = 7
+
+  // Because this enum is exported to Objective-C,
+  // the usual "\(UpdatesErrorCode.NoUpdatesAvailable)"
+  // string representation will not work as expected,
+  // so we add this representation here
+  public var asString: String {
```

Comment:
I've got an idea, maybe we can make the enum be a String instead of Int? Then I think we won't have to map it to string 🤔

Reply:
Funny story, that's exactly what I did on Android. However, I think it's better this way here as an integer enum is more usual in Objective C.